be/src/exec/pipeline/scan/morsel.cpp

```diff
@@ -851,7 +863,7 @@ Status LogicalSplitMorselQueue::_init_tablet() {
     ASSIGN_OR_RETURN(_segment_group, _create_segment_group(_largest_rowset));
 
     _short_key_schema =
-            std::make_shared<Schema>(ChunkHelper::get_short_key_schema(_tablets[_tablet_idx]->tablet_schema()));
+            std::make_shared<Schema>(ChunkHelper::get_short_key_schema(_tablet_schema));
```

Comment:
In what scenarios, the return value of `_tablets[_tablet_idx]->tablet_schema()` is nullptr?

Reply:
cache miss and then network failure, or tablet has been dropped by catalogBin